package.json

```diff
@@ -8,11 +10,26 @@
     "dev": "node . --unhandled-rejections=strict",
     "build": "next build",
     "start": "NODE_ENV=production node . --unhandled-rejections=strict",
-    "build-system": "rollup -c"
```

Comment:
Please add this back. `"build-system": "rollup -c"`

Reply:
All set w/ latest commit.